bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -32,12 +37,48 @@ object VoTableParser extends VoTableParser {
   val UCD_MAG        = UcdWord("phot.mag")
   val STAT_ERR       = UcdWord("stat.error")
 
-  private def validate(catalogName: CatalogName, xmlText: String): CatalogProblem \/ Unit =
+  private val VoTableHeader: Regex =
+    """<VOTABLE([^>]*)>""".r.unanchored
+
+  private val MinVersion: VersionToken = VersionToken.unsafeFromIntegers(1, 2)
+  private val MaxVersion: VersionToken = VersionToken.unsafeFromIntegers(1, 3)
+
+  private val VersionRegex: Regex =
+    """version="(\d+\.\d+)"""".r.unanchored
```

Comment:
Regex parsing is kind of low rent but I didn't want to dig up an old OSGi-safe version of Atto or something.

Reply:
I want an editor plugin that lets me click on a regex and it takes me to regexper, like [this](https://regexper.com/#version%3D%22%28%5Cd%2B%5C.%5Cd%2B%29%22).

Reply:
Wow, that's excellent.